cv/lane_detection/src/line_fitting.py

```diff
@@ -51,99 +49,50 @@ def lane_fitting(points):
         sigma_x = np.sqrt(np.sum(np.power(group[:,1]-x_avg,2))/group.shape[0])
         sigma_y = np.sqrt(np.sum(np.power(group[:,0]-y_avg,2))/group.shape[0])
 
-        # print(sigma_x, sigma_y)
-
         if (sigma_x < 5) and (sigma_y < 5):
             fit_points.append([y_avg,x_avg])
 
     if len(fit_points) <= SPLINE_DIM:
         return None
 
     fit_points = np.array(fit_points)
-    # print(fit_points)
 
     x = fit_points[:,1]
     y = fit_points[:,0]
     tck,u = interpolate.splprep([x,y],k=SPLINE_DIM,s=32)
-    # print(tck)
-    out = interpolate.splev(u,tck)
+
+    # u = np.linspace(u[0]-EXTRAPOLATE_VALUE,u[-1]+EXTRAPOLATE_VALUE,500)
+    out = interpolate.splev(u,tck)  # out is an array in the form of [[x_points], [y_points]]
 
     if type(out) == list:
-        return [out[0].tolist(), out[1].tolist()]
+        return np.array(out).T.tolist()
     else:
-        return out.tolist()
+        raise NotImplementedError()
+        return out.tolist() # We shouldn't be hitting this, right? 
```

Comment:
Exciting

You may also not want to raise and error and instead just return (@spencerteetaert please confirm what happens if we enter this branch). I think this will take down the process, which we don't want